backwardcompatibilityml/metrics.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import torch
+from sklearn.metrics import accuracy_score
+
+
+def model_accuracy(model, dataset, device="cpu"):
```

Comment:
The one tradeoff with this approach is that the user will have to wrap their performance function to allow its function signature to be the same as this, and pass the wrapped function in.

Could we document this somewhere?

Two places I can think of are:

1. In the docstring of the compatibility_sweeps function.
2. Another place would be on the widget so that it is documented what type of function the user passes in.

Reply:
I added docs to the compatibility_sweep function and to the CompatibilityAnalysis constructor. Here's what a user will see if they run `?CompatibilityAnalysis` in their notebook. Do you think it's sufficient? I'm not completely sure where to put this information on the widget itself without adding a lot of clutter.

![image](https://user-images.githubusercontent.com/1587757/97036772-2b529780-151d-11eb-93e4-d54395d35c1f.png)